src/org/labkey/test/pages/core/admin/AuditLogMaintenancePage.java

```diff
@@ -0,0 +1,83 @@
+package org.labkey.test.pages.core.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class AuditLogMaintenancePage extends LabKeyPage<AuditLogMaintenancePage.ElementCache>
+{
+    public AuditLogMaintenancePage(WebDriver driver)
+    {
+        super(driver);
+        waitForPage();
```

Comment:
The `LabKeyPage` default constructor calls `waitForPage`, no need to call it again here.
```suggestion
        super(driver);
```

Reply:
Done